docs/source/flow-project.rst

```diff
@@ -128,6 +128,7 @@ The entirety of the code is as follows:
         return job.isfile("hello.txt")
 
 
+    # The condition must appear after (above) the operation decorator.
```

Comment:
I didn't know enough about decorators to know how to make sense of the fact that "after" in terms of decorators means "above" in terms of lines of code.

Can we keep the explanation in terms of lines of code?

Reply:
Are you saying not to use after at all or just to ensure we mention the above/below with it. I think it is important to mention both as the order of execution of decorators in Python gives the intuition for how this works, while the above/below language helps new Python users.

Reply:
I agree we should keep both. I think being more verbose will help. How about this? Also should apply to the other change in the PR.
```suggestion
    # Pre/post condition decorators must appear on a line above the operation decorator
    # so that the condition decorator is added after the operation decorator.
```

Reply:
Oh, this is what you meant. What do you think of an aside about how decorators execute and perhaps a link? https://docs.python.org/3/reference/compound_stmts.html#function-definitions

Reply:
> aside about how decorators execute
That's good by me.

The link to the syntax is very deep for most users. Do you know of a more practical one?

Reply:
I would want to use an official source rather than a blog or something else, so this is the best one I know of.